### ISSUE TITLE
Proposal: fixed numbers of digits after decimal point

### ISSUE THREAD

**Issue by tychsen** [sparrowwallet/sparrow#114]
Proposal: Balance and transaction values presented as if it was a table / spread sheet with fixed number of decimals (propose: 8).

As digits after decimal point varies a lot show always a fixed amount (8). Gives much better visual understanding of the size of different payments.

![align](https://user-images.githubusercontent.com/42609900/116673722-08a9ab80-a9a4-11eb-8bcc-a4bea114e4a7.png)

**Comment by craigraw** [comment]
I like this idea, although it's worth noting it's not a fixed width font so the decimal point won't line up (as you have depicted).

That said, it's still easier to read - except when the value is zero (common on the addresses table) in case I think the current method is better:

<img width="207" alt="Screenshot 2021-04-30 at 16 33 22" src="https://user-images.githubusercontent.com/862166/116710429-0c542700-a9d2-11eb-9ac7-ca9669b24753.png"><img width="194" alt="Screenshot 2021-04-30 at 16 34 27" src="https://user-images.githubusercontent.com/862166/116710443-0fe7ae00-a9d2-11eb-8733-8cc6192a95b8.png">

So, perhaps a mix of both is good depending on value.

**Comment by Transisto** [comment]
I like showing 8 digits everywhere possible, steps closer to the sat standard.

Also makes it easier to show people that they're sending round amounts (ending in 00) and giving up some privacy.

**Comment by tychsen** [comment]
> I like this idea, although it's worth noting it's not a fixed width font so the decimal point won't line up (as you have depicted).

I propose using a monospace font for all number representations. I think you are using Arial, and there exist a monospaced version of this one (https://www.myfonts.com/fonts/mti/arial-monospaced-mt/).

We would achieve what we want without introducing new fonts in the application.

If not possible I would choose the other font you use in the app (for transactions and addresses) - it seems to me that it is monospaced.

So basically I see this proposal as a flawless upgrade to the UI with no drawdowns.


> That said, it's still easier to read - except when the value is zero (common on the addresses table) in case I think the current method is better:
>
> So, perhaps a mix of both is good depending on value.

I agree - 0 number representations could be with 0 decimals. Fine.

**Comment by craigraw** [comment]
Implemented in 020ca2f.

It turns out that only OSX (AFAIK) uses a system font which doesn't have fixed width spacing for numbers, so Helvetica Neue is now preferred there for BTC amounts in tables.

<img width="343" alt="Screenshot 2021-05-04 at 10 26 03" src="https://user-images.githubusercontent.com/862166/116977813-29834100-acc3-11eb-8892-892b8bfd7cb4.png">
